### ISSUE TITLE
Increase space between bottomsheet and buttons in navigation

### ISSUE THREAD

**Issue by Jean-BaptisteC** [organicmaps/organicmaps#7085]
**Steps to reproduce**
1. Open Organic Maps in the Arabic language
2. Select POI and touch route to
3. Start routing
4. Rotate device in landscape
5. Space between bottomsheet and buttons is too weak

**Screenshots**
<img src="https://github.com/organicmaps/organicmaps/assets/87148630/602e531b-4353-4115-aaec-177464c2fe9f" height=200 />

**System information:**
 - Operating system and its version: Android 14
 - Organic Maps version: 2023.12.21-1-Google
 - Device Model: Pixel 6

No problem in french, I guess arabic text take a little more space

Correction need to be work on LTR and RTL

**Comment by rohit9625** [comment]
Hello @Jean-BaptisteC 
I was trying to build the Android app but got this error :
`
Build file 'D:\GitHub\organicmaps\android\app\build.gradle' line: 64

A problem occurred evaluating project ':app'.
> Could not read script 'D:\GitHub\organicmaps\android\app\secure.properties' as it does not exist.
`

I followed all the steps provided in the docs specifically for Android build.

**Comment by Jean-BaptisteC** [comment]
Please follow this part https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md#getting-sources

**Comment by rohit9625** [comment]
Thank you @Jean-BaptisteC 
Now I can build the app. How can I open the app in Arabic language is there any in app settings or should I change the language from settings?

**Comment by sumitgarg21** [comment]
I want to work on it. please assign this to me

**Comment by Jean-BaptisteC** [comment]
@rohit9625 Yes you need to change the system language, we don't have not yet implemented app language

**Comment by adysinghh** [comment]
@Jean-BaptisteC I have created a PR for this issue please review

**Comment by kavikhalique** [comment]
I have checked this issue in main OM app which is on playstore but there is no such issue, but in debug OM this issue is present. I guess issue is in Debug OM app.

**Comment by biodranik** [comment]
How is it possible that the issue reproduces only in debug?

**Comment by Jean-BaptisteC** [comment]
Yes only on debug, not in release version

**Comment by biodranik** [comment]
How is it possible that debug and release have different insets❓ I don't have any ideas

**Comment by Jean-BaptisteC** [comment]
It seems size of top margin of bottomsheet is not everytime the same.

**Comment by kavikhalique** [comment]
> It seems size of top margin of bottomsheet is not everytime the same.

I feel the issue is due to the change in font. Height of buttons above bottom sheet are fixed to 70dp and size of bottom sheet is changing because of font change. when any character of arabic which have lower side expansion, comes, overall layout size increases and since layout  is wrap content, its size increases which sometimes results in overlap.
